airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -121,20 +123,20 @@ def parse_response(self, response: requests.Response, stream_state: Mapping[str,
                 if not cursor_date or updated > cursor_date:
                     yield record
 
-    def should_retry(self, response: requests.Response) -> bool:
-        status_code = response.status_code
-        if status_code == 403 or status_code == 404:
-            try:
-                error = response.json().get("error")
-            except requests.exceptions.JSONDecodeError:
-                reason = response.reason
-                error = {"title": f"{reason}", "message": "Received empty JSON response"}
-            self.logger.error(
-                f"Skipping stream {self.name}, error message: {error}. Please ensure the authenticated user has access to this stream. If the issue persists, contact Zendesk support."
-            )
-            setattr(self, "raise_on_http_errors", False)
-            return False
-        return super().should_retry(response)
+    def get_error_handler(self) -> Optional[ErrorHandler]:
+        error_mapping = DEFAULT_ERROR_MAPPING | {
+            403: ErrorResolution(
+                response_action=ResponseAction.IGNORE,
+                failure_type=FailureType.config_error,
+                error_message="Forbidden. Please ensure the authenticated user has access to this stream. If the issue persists, contact Zendesk support.",
+            ),
+            404: ErrorResolution(
+                response_action=ResponseAction.IGNORE,
+                failure_type=FailureType.config_error,
+                error_message="Not found. Please ensure the authenticated user has access to this stream. If the issue persists, contact Zendesk support.",
+            ),
+        }
+        return ZendeskSupportHttpStatusErrorHandler(logger=self.logger, max_retries=10, error_mapping=error_mapping)
```

Comment:
Can we instantiate from `HttpStatusErrorHandler` directly, instead?

Reply:
agreed, ref